ios/Video/RCTVideo-Bridging-Header.h

```diff
@@ -1,6 +1,6 @@
 #import "RCTVideoSwiftLog.h"
 #import <React/RCTViewManager.h>
-
+#import <React/RCTEventDispatcher.h>
```

Comment:
I think this is added to fix a build issue.
We saw some duplicated import by adding this line here.
It would be better to remove it ...